lambdas/service/app.py

```diff
@@ -769,13 +806,67 @@ def files_proxy(uuid):
         dss_response.raise_for_status()
 
 
+def self_url(endpoint_path=None):
```

Comment:
This method already exists on `develop`.

Reply:
The duplicate has been removed.